libass/ass_priv.h

```diff
@@ -65,6 +65,10 @@ struct parser_priv {
     uint32_t header_flags;
 
     uint32_t feature_flags;
+
+    int gc_enabled;
+    long long gc_delay;
+    long long gc_next_undisplay_ts;
```

Comment:
I think `gc_enabled` and `gc_delay` should be merged. Also if we're moving form GC to prune, it should be renamed too:
```c
int prune_enabled;
long long prune_delay;
long long prune_next_ts;
```